src/App.js

```diff
@@ -1,9 +1,10 @@
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
```

Comment:
- [ ] It would be better if your `App` component is class-based as one of the requirements. Kindly fix this minor issue. :+1: :smile: